src/main/templates/partials/header.hbs

```diff
@@ -0,0 +1,38 @@
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+    integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+  <link rel="stylesheet" type="text/css" href="mobile.css">
+  <title>ISDCF {{listTitle}}</title>
+  <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js" crossorigin="anonymous"></script>
+  <script type="application/javascript" src="features.js"></script>
+</head>
+
+<body>
+  <nav class="navbar navbar-expand-md navbar-light fixed-top bg-light d-print-none d-flex">
+    <a class="navbar-brand" href="#">ISDCF {{listTitle}}</a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarNav">
+      <ul class="navbar-nav mr-auto">
+        <li class="nav-item">
+          <a id="json-feed" target="_blank" class="nav-link" href="https://registry.isdcf.com/{{listType}}">JSON Feed</a>
+        </li>
+        <li class="nav-item">
+          <a id="pdf-download" target="_blank" class="nav-link" title="Download as PDF" href="{{pdf_path}}">Download PDF</a>
+        </li>
+      </ul>
+        <form id="search-form" class="form-inline">
+          <span class="align-middle"><input id="search" class="form-control mb-2" type="text" placeholder="Search {{listTitle}}">
+          <input type="button" class="btn btn-secondary btn-sm mb-2 clear-filter" value="x"/></span>
+        </form>
+    </div>
+  </nav>
+  <a href="#" id="toTopBtn" class="cd-top text-replace js-cd-top cd-top--is-visible cd-top--fade-out" data-abc="true"></a>
+  <a class="anchor" id="top" href="#top"></a>
+  <div class="d-none d-print-block"><h4>ISDCF {{listTitle}}</h4></div>
```

Comment:
What is the purpose of this `h4`?

Reply:
Strictly for the print version.

Reply:
The print version work correctly today, so what is the improvement. We should not have two headers with the same content/semantics.

![image](https://user-images.githubusercontent.com/4871350/88434882-f451d600-cdb5-11ea-8a58-76ea1bad89b5.png)

Reply:
The structure of the Nav bar and ability have the collapse feature (for mobile) was not working properly with the fixed-top for the Nav without this. When applying the d-print-none class to the div navbarNav (the menu), it doesn't actually hide the menu, and the headers get chopped. See below. 

<img width="803" alt="Screen Shot 2020-07-24 at 2 13 50 PM" src="https://user-images.githubusercontent.com/38917682/88435840-edc45e00-cdb7-11ea-9c2f-912c73b135cc.png">

Without the fixed top, the page becomes very unusable quickly (esp on mobile) the second you scroll down (esp if you have a current search filter on the page). This was the best workaround I could get working cross platform.

Reply:
Oh. This is already in the code. Will file a separate issue.